feature1/one-more-file.txt

```diff
@@ -0,0 +1,3 @@
+This file was added also in the feature1 branch
```

Comment:
It is really good that you have decided to add that line of code

Reply:
Thanks for this comment